nix/api.nix

```diff
@@ -12,15 +12,14 @@ rec {
 	resolve = {
 		ocaml,
 		selection,
-		# NOTE: these aren't used, but are explicitly allowed for compatibility with `build` / `buildInputs`
-		override ? null,
-		builtinOverride ? null,
-		src ? null,
+		repo ? null,
+		...
```

Comment:
It's a bit silly looking, but I don't want to actually ignore all arguments, because that gives bad feedback on e.g typos. I'm not even sure allowing these existing ones was a good idea, but it's not the time to roll that back ;)